src/SecurityDetails/SecurityDetailsJSON.ts

```diff
@@ -58,6 +58,52 @@ namespace SecurityDetailsJSON {
         ActualManagementFee: number;
         FundAttributes?: FundAttributesType;
         HistoricalPerformanceSeries?: SecurityDetailsHistoricalPerformance[];
+        RiskStatistics?: RiskStatisticsItem[];
+    }
+
+    interface RiskStatisticsItem {
```

Comment:
Missing `Correlations` and `BenchmarkCorrelations`:

<img width="355" height="280" alt="Image" src="https://github.com/user-attachments/assets/080c04f8-5476-4e6d-9c49-f8787df03cf7" />